cmd/utils/flags.go

```diff
@@ -925,11 +925,19 @@ func MakeChain(ctx *cli.Context, stack *node.Node) (chain *core.BlockChain, chai
 	chainDb = MakeChainDatabase(ctx, stack)
 
 	if ctx.GlobalBool(OlympicFlag.Name) {
+		_, err := core.WriteOlympicGenesisBlock(chainDb)
+		if err != nil {
+			glog.Fatalln(err)
+		}
+	}
+
+	if ctx.GlobalBool(TestNetFlag.Name) {
```

Comment:
Are we sure this is correct? Didn't this keep `--olympic` and drop `--testnet` (although `--olympic` would now load ropsten)?